haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Client for LiveJournal online journalling community"
```

Comment:
Oops, I had not paid enough attention to the SUMMARY.
I think this would be better.
```diff
-SUMMARY="Client for LiveJournal online journalling community"
+SUMMARY="A client for the LiveJournal online journalling community"
```
The README has this sentence:
> AliveJournal is a client for the LiveJournal online journalling community.

BTW, did you know that haikuporter refuses to process recipes if the SUMMARY starts with the name of the package?
So if you ever start a draft recipe for a program, say foobar, and define SUMMARY="foobar bla bla bla" it will not work :D

Reply:
Wait, really? I did not know that.